Grasshopper_UI/Components/Adapter/Push.cs

```diff
@@ -45,7 +45,7 @@ public class PushComponent : CallerComponent
         protected override void RegisterInputParams(GH_InputParamManager pManager = null)
         {
             base.RegisterInputParams(pManager);
-            if (Params.Input.Count > 1)
+            if (pManager != null && Params.Input.Count > 1)
```

Comment:
I believe it shoudl be the opposite
```suggestion
            if (Params.Input.Count > 1)
                params.Input[1].DataMapping  = GH_DataMapping.Flatten;
```